crates/varcon/assets/README

```diff
@@ -220,7 +220,7 @@ created by Garst R. Reese <reese@isn.net>:
   Canadians use ize ization, izing izable like Americans, and gram instead
   of gramme. The one exception I found was practise. It does not go to
   practize.  Otherwise they use British spelling. So, what I am currently
-  checking books with is a an edited version of British, where I have
+  checking books with is an edited version of British, where I have
```

Comment:
Please revert this as its a vendored file

Reply:
reverted